backend/app/tests/test_guardrails_api.py

```
@@ -18,20 +19,19 @@ def mock_crud():
 
 
-def test_routes_exist(client):
+def test_route_exists(client):
     paths = {route.path for route in client.app.routes}
-    assert "/api/v1/guardrails/input/" in paths
-    assert "/api/v1/guardrails/output/" in paths
+    assert VALIDATE_API_PATH in paths
 
 
-def test_input_guardrails_success(client, mock_crud):
+def test_validate_guardrails_success(client, mock_crud):
```

Ruff (0.14.14)

Unused function argument: `mock_crud`

(ARG001)